requirements.txt

```diff
@@ -6,3 +6,4 @@ uwsgi==2.0.10
 psycopg2==2.6
 tox==1.9.2
 static3==0.5.1
+git+https://github.com/mitodl/xbundle.git
```

Comment:
Looks like we are still getting the beautiful soup stuff installed, maybe try changing this to:
`git+https://github.com/mitodl/xbundle.git@d6eb740e023f52f726b8830990ac108df547b0a8#egg=xbundle`
to make sure nothing is cached